src/sentry/middleware/integrations/parsers/base.py

```diff
@@ -83,8 +84,11 @@ def get_response_from_control_silo(self) -> HttpResponseBase:
         return self.response_handler(self.request)
 
     def get_response_from_region_silo(self, region: Region) -> HttpResponseBase:
-        region_client = RegionSiloClient(region)
-        return region_client.proxy_request(incoming_request=self.request)
+        with metrics.timer(
+            "integration_proxy.control.get_response_from_region_silo", sample_rate=1.0
```

Comment:
We'll likely need to revert the sample_rate once we start handling production traffic volumes.